app/routes/articles/components/Overview.js

```diff
@@ -80,12 +80,17 @@ export default class Overview extends Component<Props> {
               <Tag
                 tag={tag.tag}
                 key={tag.tag}
-                color={isSelected ? 'red' : ''}
+                color={isSelected ? 'purple' : 'blue'}
```

Comment:
Hmmmm, I am not sure about this choice of colours, blue and purple does not fit with our theme imo

Reply:
Abakus don't really have a color palette besides red and gray, sadly, and I think we should be more "open-minded" with introducing more colors here and there. Nevertheless, I'm okay with changing these specific colors to something else though, as I'm not sure if I like them either. Any suggestions?

Reply:
It is now solely blue, and instead employing a `active` prop to give feedback.